src/rust/en.tsumino/src/parser.rs

```diff
@@ -0,0 +1,84 @@
+use crate::helper::get_id;
+use aidoku::{
+	error::Result,
+	prelude::*,
+	std::ObjectRef,
+	std::String,
+	std::{html::Node, Vec},
+	Manga, MangaContentRating, MangaStatus, MangaViewer,
+};
+
+pub fn parse_list(manga_obj: ObjectRef) -> Result<Manga> {
+	let main = manga_obj.get("entry").as_object()?;
+	let id = get_id(main.get("id"))?;
+	let title = main.get("title").as_string()?.read();
+	let cover = main.get("thumbnailUrl").as_string()?.read();
```

Comment:
is this always defined? might be better to unwrap to an empty string if it isn't